packages/web/pages/api/feed.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import NextCors from 'nextjs-cors';
+import { parse } from 'rss-to-json';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  // Run the cors middleware
+  // nextjs-cors uses the cors package, so we invite you to check the documentation https://github.com/expressjs/cors
+  await NextCors(req, res, {
```

Comment:
Is this package necessary? Aren't these just GET requests?

Reply:
It was also used in some of the examples. 
But how does it work without using nextjs-cors package ? we are making a request from a different domain